### PR TITLE
Check for a data-video-id option as a valid default source

### PR DESCRIPTION
**Pull request by bc-bbay**
Don’t throw an error if we find one

### CODE REVIEW

**Comment by bclwhitaker** [comment]
LGTM

File: src/videojs.errors.js [hunk 1]
```diff
@@ -139,9 +139,9 @@
 
     // PLAYER_ERR_NO_SRC
     player.on('play', function() {
-      if (player.currentSrc() === null ||
+      if (!player.options()['data-video-id'] && (player.currentSrc() === null ||
```
Comment (bclwhitaker) [inline src/videojs.errors.js]: do we also need to check for playlist id?

**Comment by dmlap** [comment]
`data-video-id` is not a standard attribute. What problem is this PR trying to address?

**Comment by bc-bbay** [comment]
Another solution was found. No need to add BC code to an open source project.

**Pull request closed without merging**